modules/custom_operations/README.md

```diff
@@ -0,0 +1,50 @@
+# Collection of Custom Operations using OpenVINO Extensibility Mechanism
+
+This module provides a guide and the implementation a few custom operations in the Intel OpenVINO runtime using its [Extensibility Mechanism](https://docs.openvino.ai/latest/openvino_docs_Extensibility_UG_Intro.html).
+
```

Comment:
We haven't covered the extensibility story for Pytorch in the documentation. We have this document https://docs.openvino.ai/latest/openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch.html that is basically a forward to `torch.onnx.export` tutorial in Pytorch documentation, and a couple of specific examples that are not illustrating custom ops.

As we haven't yet provided direct Pytorch importing then for the user this story breaks up into two steps: express it in ONNX first, and then import ONNX to the runtime. For the first part, we are not providing much help except mentioned `torch.onnx.export` link which is covering custom ops expression in ONNX.

With `openvino.tools.mo.convert_model` which hides `torch.onnx.export` underneath the story should be a bit more elegant, because we will try to export all ops in ONNX with `ONNX_FALLTHROUGH` regardless of support in ONNX default opset, and will be able to intercept custom ops in our ONNX FE as usual for other ONNX custom ops via extension. Currently, we are not doing that, but I manually tried to modify _a single line of code_ in the `convert_model` implementation and it worked for the `sparse_conv` exported model from this PR. So it is doable without explaining `torch.onnx.export` parameters and better to focus on `openvino.tools.mo.convert_model` polishing to avoid such details in our documentation.

Reply:
I agree with @yury-gorbachev: I'm missing some explanation describing what is expected from the pytorch level operation implementation, and how that needs to be propagated to ONNX. It seems to involve implementation of the operation by deriving from `torch.autograd.Function` and overriding the `forward()` and `symbolic()` methods. What is expected from the `symbolic()` function? Do we need to use `ONNX_PASSTHROUGH`, or `ATEN`, or does it not matter? 

Perhaps what we're missing is an example of a pytorch model with only native pytorch ops, and how to create a custom operation for it since it isn't supported by OpenVINO? I thought the FFT example was that, but it seems that it does more than just add a custom operation for `torch.fft`. Should we simplify it?